### ISSUE TITLE
accessibility warning in setTimeout breaks tests 

### ISSUE THREAD

**Issue by jnizet** [highcharts/highcharts#17236]
Version 10.1.0 introduced an error message in case the accessibility module is not loaded. See the change here: https://github.com/highcharts/highcharts-dist/blob/master/es-modules/Core/Chart/Chart.js#L1953-L1966

Unfortunately, it does that in a setTimeout, and there is no way to disable this setTimeout, even by disabling accessibility or adding the accessibility module, because the checks are done in the setTimeout callback itself. So, this setTimeout is always executed, no matter what.
It might not seem like a big deal, but we have tests (in an Angular application), using a "fakeAsync" zone where tests fail shen a  timer is still in the queue. And this timeout does that: it introduces a new, unexpected timer, in the queue. 

#### Product version
Highcharts 10.1.0

#### Affected browser(s)
All

**Comment by oysteinmoseng** [comment]
Hi @jnizet, interesting use case.

A workaround could be to override the warning function before creating the charts in the tests: `Highcharts.Chart.prototype.warnIfA11yModuleNotLoaded = () => {};` [Live demo](https://jsfiddle.net/apxz24v5/).

We will also look into if there is a robust way to handle this on our end without using timers.

**Comment by jnizet** [comment]
Hi @oysteinmoseng. Thanks. That's exactly the workaround we used. But it's a dirty hack, especially with TypeScript which doesn't know about the `warnIfA11yModuleNotLoaded` property.
Thanks for investigating a cleaner way.

**Comment by jnizet** [comment]
Thanks for the fix @TorsteinHonsi !
